### PR TITLE
Styling updates

### PR DESCRIPTION
**Pull request by rowanhogan**
- Adds option for 'Block' style layout
- Adds appropriate max widths more in line with mockup
- Better body copy line lengths!

### CODE REVIEW

File: source/components/Search/styles.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const container = {
 
 export const form = {
   maxWidth: rhythm(32),
-  padding: rhythm(1),
+  // padding: rhythm(1),
```
Comment (ellsclytn) [inline source/components/Search/styles.js]: Do we still need this?

File: source/components/StructuredText/styles.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+export const content = (theme, align) => ({
+  ...fonts('body', theme),
   margin: `${rhythm(1)} auto`,
+  maxWidth: '36em',
+  textAlign: align || 'center',
   lineHeight: '1.45em',
-  ...fonts('body', theme),
+  WebkitHyphens: 'auto',
+  MsHyphens: 'auto',
+  hyphens: 'auto',
```
Comment (ellsclytn) [inline source/components/StructuredText/styles.js]: No autoprefixer support?

File: source/components/Image/styles.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
-import { rhythm } from '../../lib/traits'
-
 export default {
   wrapper: {
-    maxWidth: rhythm(32),
-    padding: rhythm(1),
-    margin: 'auto'
+    padding: 0,
+    '@media (min-width: 0)': {
```
Comment (ellsclytn) [inline source/components/Image/styles.js]: This appears redundant. Or is this to override a persisting media query?

Comment (rowanhogan) [inline source/components/Image/styles.js]: Yes, unfortunately it's required to override an existing media query (as that is more specific). However I probably don't need to set the padding outside the query...

**Pull request merged**